### ISSUE TITLE
Analyzer: Falsely matches constructor

### ISSUE THREAD

**Issue by ardalis** [ardalis/ApiEndpoints#29]
Repro:
Use the sample project and look at any of the endpoints there.

![image](https://user-images.githubusercontent.com/782127/89671517-1fa5ec00-d8b1-11ea-89d9-be9e50734308.png)

Constructors should be ignored by the analyzer.

**Comment by ardalis** [comment]
@ppittle do you want to look at this?

**Comment by fiseni** [comment]
This did the trick.

```

                if (methodSymbol.MethodKind == MethodKind.Constructor)
                    return;
```

**Comment by ppittle** [comment]
Ooph, good catch.  Saw that @fiseni code is already in master, so I added a regression test as part of #31

**Comment by ardalis** [comment]
Looks like this is fixed by #31 Thanks!
